templates/base.html

```diff
@@ -48,23 +85,28 @@
                     {% block content %}{% endblock %}
                 </div>
             </div>
-            <footer><small>
-                <hr>
-                <a href="https://twitter.com/happinesspacket" class="twitter-follow-button" data-show-count="false" data-size="large" data-dnt="true">Follow @happinesspacket</a>
-                <script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+'://platform.twitter.com/widgets.js';fjs.parentNode.insertBefore(js,fjs);}}(document, 'script', 'twitter-wjs');</script>
-                <br>
-                Open-Source Happiness Packets is an
-                <a href="https://github.com/mxsasha/happinesspackets/">open-source project</a> by
-                <a href="https://twitter.com/mxsash">Sasha Romijn</a> and
-                <a href="https://twitter.com/thatdocslady">Mikey Ariel</a>.
-                <br>
-                Design and artwork by <a href="https://twitter.com/olasitarska">Ola Sitarska</a>.
-                <br>
-                Need help? <a href="mailto:info@happinesspackets.io">info@happinesspackets.io</a>
-                <img class="emoji" src="{% static 'images/emoji/loveletter.png' %}" alt="💌" title="Love letter" aria-label="Emoji: Love letter" style="vertical-align: text-bottom">
-
-            </small></footer>
         </div>
-    </main>
-</body>
+            <footer class="footer">
+              <small>
+                  <hr>
+                  {% block footer %}
+                  {% include "follow.html" %}
+                  <div class="footer-msg">
+                    {% endblock %}
+                    Fedora Happiness Packets is a fork of <a href="https://github.com/mxsasha/happinesspackets/">Happiness Packets</a> and part of GSoC 2018.
```

Comment:
This is not the Fedora Happiness Packets project though, it is the upstream Happiness Packets project :slightly_smiling_face: